Unix/mak/rules.mak

```diff
@@ -176,6 +180,15 @@ CXXPROGFLAGS+=$(shell $(BUILDTOOL) syslibs)
 CXXPROGFLAGS+=$(OPENSSL_LIBS)
 CXXPROGFLAGS+=$(LIBPATHFLAGS)
 
+ifeq ($(ENABLE_GCOV),1)
+  CFLAGS+=-lgcov
+  CPROGFLAGS+=-lgcov
+  CSHLIBFLAGS+=-lgcov
+  CXXFLAGS+=-lgcov
+  CXXPROGFLAGS+=-lgcov
+  CXXSHLIBFLAGS+=-lgcov
+endif
```

Comment:
Fixed, and addition information why I add -lgcov in the other 4 flags:
```
If I don't add -lgcov in CSHLIBFLAGS and CXXSHLIBFLAGS, it will fail with below error:

/usr/bin/ld: /test/omi/Unix/output/bin/nits: hidden symbol `__gcov_init' in /usr/lib/gcc/x86_64-redhat-linux/4.8.5/libgcov.a(_gcov.o) is referenced by DSO
/usr/bin/ld: final link failed: Bad value
collect2: error: ld returned 1 exit status


If I don't add -lgcov in CPROGFLAGS and CXXPROGFLAGS, it will fail with below error:

g++ -o /test/omi/Unix/output/bin/chkshlib /test/omi/Unix/output/obj/chkshlib/chkshlib.o -L/test/omi/Unix/output/lib -L/usr/lib64      -lpthread -ldl -lpam -lssl -lcrypto    -Wl,-rpath=/opt/omi-1.0.8/lib -Wl,-rpath=/usr/lib64
/test/omi/Unix/output/obj/chkshlib/chkshlib.o: In function `_GLOBAL__sub_I_65535_0_chkshlib.cpp':
chkshlib.cpp:(.text.startup+0xd0): undefined reference to `__gcov_init'
/test/omi/Unix/output/obj/chkshlib/chkshlib.o:(.data.rel+0x20): undefined reference to `__gcov_merge_add'
collect2: error: ld returned 1 exit status
```